installer/targets/ros/install.bash

```diff
@@ -8,15 +8,15 @@ if [ ! -d /opt/ros/$TUE_ROS_DISTRO ]
 then
 
     sudo apt-get install lsb wget -y
-    
+
     sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 
     wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
 
-    sudo apt-get update
+    sudo apt-get update -qq
 
     # Install basic ROS packages. All other packages will be installed using tue-rosdep
-    sudo apt-get install  --assume-yes ros-$TUE_ROS_DISTRO-ros-base cmake python-catkin-pkg python-empy python-nose python-setuptools libgtest-dev build-essential python-catkin-tools
```

Comment:
Are you sure that we don't need python-setuptools for the setup.py files? And python-nose for test?

Reply:
python-setuptools and python-nose are dependencies of the rest, so these will be installed. I don't know if replacing ros-base with ros is ok. We need to test that with a clean install.